tests/TimeSpanTest.php

```diff
@@ -208,6 +208,7 @@ public function testConstructorWithoutArgs(): void
     #[TestWith([100.999_99, 101])]
     #[TestWith([PHP_INT_MAX, PHP_INT_MAX])]
     #[TestWith([PHP_INT_MIN, PHP_INT_MIN])]
+    #[TestWith([PHP_INT_MIN - 1_024, PHP_INT_MIN])]
```

Comment:
This should not work. If we pass a number < PHP_INT_MIN, there should be an exception.